omero/developers/Java.rst

```diff
@@ -990,6 +990,63 @@ in OMERO. Similar approach can be applied when uploading an image.
         store.close();
     }
 
+Sudo (working within another user's context)
+--------------------------------------------
+
+The next example shows how one can work within another user's context. For example
```

Comment:
I feel like these first two sentences could be linked differently for the better; @hflynn may have a suggestion.